index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Norman life</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css"
+    integrity="sha512-8bHTC73gkZ7rZ7vpqUQThUDhqcNFyYi2xgDgPDHc+GXVGHXq+xPjynxIopALmOPqzo9JZj0k6OqqewdGO3EsrQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <div id="main">
+    <div class="page-wrapper">
+      <div id="title" class="ui huge header p-header">
+        <h1>Dr. Norman Borlaug</h1>
+        <p class="sub header">The man who saved a billion lives</p>
+      </div>
+      <div id="img-div" class="hero-container">
+        <img id="image" class="ui massive centered image" src="images/hero.jpg" alt="cover-imager" />
+        <p id="img-caption" class="img-caption">Dr. Norman Borlaug, third from the right, trains biologists in
+          Mexico on how to increase the wheat yields - part of his life-long war on hunger.</p>
+      </div>
+
+
```

Comment:
I think it would be better to remove blank spaces. it makes  your codes look better